.github/workflows/ci.yml

```
@@ -1,21 +1,81 @@
-    branches: [main]
+    branches: [main, release/1.0]
   workflow_dispatch:
 
 permissions:
   contents: read
 
 jobs:
-  build:
-    name: Lint and Build
+  # Native App Checks (Release 1.0 Focus)
+  native-checks:
```

actionlint (1.7.10)

"steps" section is missing in job "native-checks"

(syntax-check)